pandas/core/algorithms.py

```diff
@@ -325,15 +325,14 @@ def unique(values):
 
     Returns
     -------
-    numpy.ndarray or ExtensionArray
-
-        The return can be:
-
-        * Index : when the input is an Index
-        * Categorical : when the input is a Categorical dtype
-        * ndarray : when the input is a Series/ndarray
-
-        Return numpy.ndarray or ExtensionArray.
+    The return can be:
+
+    * Index : when the input is an Index
+    * ndarray : when the input is a Series/ndarray
+    * If the argument is an :ref:` ExtensionArray <extending.extension-types>`,
+    the return type matches argument type. In pandas, classes implementing the
```

Comment:
Could you remove the `In pandas, ...` sentence? It's going to be difficult maintaining the types that apply here.